opm/simulators/timestepping/SimulatorReport.hpp

```diff
@@ -46,23 +47,31 @@ namespace Opm
         bool converged;
         int exit_status;
 
+	double global_time;
+        double timestep_length;
```

Comment:
I'm sure there's a rule somewhere that every PR that complains about formatting ends up making at least one formatting faux pas...

Reply:
Apparently I did not untabify that file. Will do and amend...